### PR TITLE
1 initial implementation

### LINKED ISSUE #1: Initial implementation

### PR DESCRIPTION
**Pull request by aliaminibagh**
Hi, first version of API is ready.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,112 @@
...
+
+## **Direct Inference**
+
+In this section we will describe how to perform inference directly. For using fastapi, uvicorn server is used. Steps are as follows.
+
+1) Virtual environment
+
+    Please as first step create a virtual environment. (as an example you can use conda command as follow)
+    ```sh
+    conda create -n venv_name python=3.7
```
Comment (a-sharifi) [inline README.md]: replace this with 
```
conda create -n venv_name python
```

File: main.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+import io
+import cv2
+import numpy as np
+from starlette.responses import StreamingResponse
+from fastapi import FastAPI, File, UploadFile
+from fastapi.responses import RedirectResponse
+from src import util
+from src.body import Body
+from src.hand import Hand
+
```
Comment (a-sharifi) [inline main.py:12]: need space (format)

**Review by a-sharifi — COMMENTED** [review]
Thank you for your hard-working,
Please fix the issues which I mention and then push it again.

**Comment by aliaminibagh** [comment]
> 
Sure.

**Review by a-sharifi — APPROVED** [review]

**Pull request merged**